File: JavaWebFramework/src/OpenDataAccess/Entity/RequestMethodAttribute.java
package OpenDataAccess.Entity;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface RequestMethodAttribute {
    public boolean IsGet = true;
    public boolean IsPost = true;
    public boolean IsPut = true;
    public boolean IsDelete = true;
}

File: AbetAccount/AbetAccount_Api/OpenDataAccessCore/OpenDataAccessCore.Entity/RequestMethodAttribute.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace OpenDataAccessCore.Entity
{
    public class RequestMethodAttribute : Attribute
    {
        public RequestMethodAttribute()
            : base()
        {
            this.IsDelete = true;
            this.IsGet = true;
            this.IsPost = true;
            this.IsPut = true;
        }

        public bool IsGet { get; set; }
        public bool IsPost { get; set; }
        public bool IsPut { get; set; }
        public bool IsDelete { get; set; }
    }
}
